### ISSUE TITLE
Large binary file in npm package on v2.1.1

### ISSUE THREAD

**Issue by LewisJEllis** [getsentry/raven-node#361]
A fresh `npm install` results in a `twenty-six-megs-orig` file on v2.1.1 but not v2.1.0. It's rather large and doesn't appear to serve any purpose. I'm not sure what its contents are or how it got there.

```
:[/Users/lewis/dev/sandbox]$ ls -al
total 0
drwxr-xr-x   2 lewis  staff   68 Aug  9 12:45 .
drwxr-xr-x  15 lewis  staff  510 Aug  8 10:55 ..

:[/Users/lewis/dev/sandbox]$ npm install raven@2.1.0
+ raven@2.1.0
added 7 packages in 0.477s

:[/Users/lewis/dev/sandbox]$ ls -al node_modules/raven
total 56
drwxr-xr-x   9 lewis  staff    306 Aug  9 12:45 .
drwxr-xr-x  10 lewis  staff    340 Aug  9 12:45 ..
-rw-r--r--   1 lewis  staff  10470 Jun 20 15:18 History.md
-rw-r--r--   1 lewis  staff   1289 Nov 15  2016 LICENSE
-rw-r--r--   1 lewis  staff    422 Nov 15  2016 README.md
drwxr-xr-x   3 lewis  staff    102 Aug  9 12:45 bin
-rw-r--r--   1 lewis  staff    274 May 16 17:31 index.js
drwxr-xr-x   7 lewis  staff    238 Aug  9 12:45 lib
-rw-r--r--   1 lewis  staff   2165 Aug  9 12:45 package.json

:[/Users/lewis/dev/sandbox]$ npm install raven@2.1.1
+ raven@2.1.1
updated 1 package in 0.627s

:[/Users/lewis/dev/sandbox]$ ls -al node_modules/raven
total 53304
drwxr-xr-x  10 lewis  staff       340 Aug  9 12:45 .
drwxr-xr-x  10 lewis  staff       340 Aug  9 12:45 ..
-rw-r--r--   1 lewis  staff     10589 Jul 27 15:30 History.md
-rw-r--r--   1 lewis  staff      1289 Nov  3  2015 LICENSE
-rw-r--r--   1 lewis  staff       422 Mar 26 09:47 README.md
drwxr-xr-x   3 lewis  staff       102 Aug  9 12:45 bin
-rw-r--r--   1 lewis  staff       274 Jul 14 15:22 index.js
drwxr-xr-x   7 lewis  staff       238 Aug  9 12:45 lib
-rw-r--r--   1 lewis  staff      2165 Aug  9 12:45 package.json
-rw-------   1 lewis  staff  27262976 Aug  4  2016 twenty-six-megs-orig

:[/Users/lewis/dev/sandbox]$ npm -v
5.0.3
```

**Comment by orobardet** [comment]
I just notice this file too on raven-node v2.1.1. 
In my case too it comes with a fresh install from npm, in a Docker image build. 
This 26Mb file in a node-alpine image kill the purpose of using a lightweigh alpine :)

**Comment by benvinegar** [comment]
Published 2.1.2.
